src/torque-offset-estimator.cpp

```diff
@@ -93,10 +93,14 @@ namespace dynamicgraph
       {
         try {
           se3::urdf::buildModel(urdfFile,se3::JointModelFreeFlyer(),m_model);
-          assert(m_model.nq == N_JOINTS+7);
-          assert(m_model.nv == N_JOINTS+6);
-          jointTorqueOffsets.resize(N_JOINTS);
+          // assert(m_model.nq == N_JOINTS+7);
+          // assert(m_model.nv == N_JOINTS+6);
+
+          jointTorqueOffsets.resize(m_model.nv-6);
           jointTorqueOffsets.setZero();
+
+          ffIndex = 0;
+          torsoIndex = m_model.getJointId("torso_2_joint");
```

Comment:
To be general you should take the name of the joint associated to the body containing the IMU ("torso_2_joint") as an input parameter of the init command.